census/estimator/trainer/model.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2016 Google Inc. All Rights Reserved. Licensed under the Apache
+# License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import multiprocessing
+
+import tensorflow as tf
+from tensorflow.contrib import layers
```

Comment:
I think it's pretty cluttered since layers is used tons of places. And people should be able to look up at the import. Additionally this will make migration much easier when layers moves into core.